src/templates.ts

```diff
@@ -110,16 +110,36 @@ export function getTemplates(options: ModuleOptions, uiConfig: Record<string, an
   writeThemeTemplate(theme)
 
   async function generateSources() {
-    let sources = ''
+    if (!nuxt) {
+      return '@source "./ui";'
+    }
+
+    const sources: string[] = []
+    const layers = getLayerDirectories(nuxt).map(layer => layer.app)
```

Comment:
@atinux I'm using `layer.app`, do you think it is enough or should we use `.root`?

Reply:
I will go with `layer.app` to start honestly